SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -996,19 +1021,38 @@ void USpatialReceiver::HandleUnreliableRPC(Worker_ComponentUpdateOp& Op)
 				ObjectRef.Entity = Schema_GetEntityId(EventData, SpatialConstants::UNREAL_PACKED_RPC_PAYLOAD_ENTITY_ID);
 			}
 		}
-
+		
 		UObject* TargetObject = PackageMap->GetObjectFromUnrealObjectRef(ObjectRef).Get();
 
+		// TODO(Alex): Maybe queue by ObjectRef (create jira).
```

Comment:
Why aren't we doing this? Again don't add todos to make jira tickets

Reply:
This issue is resolved now.